libs/template/templates/lakeflow-pipelines/template/{{.project_name}}/resources/{{.project_name}}_pipeline/{{.project_name}}.pipeline.yml.tmpl

```diff
@@ -0,0 +1,14 @@
+resources:
+  pipelines:
+    {{template `pipeline_name` .}}:
+      name: {{template `pipeline_name` .}}
+      serverless: true
+      continuous: false
+      channel: "PREVIEW"
```

Comment:
Should users be on preview by default?

Reply:
While we're in beta we've been using `PREVIEW`. We haven't fully validated if it works iwthout recently, but let's discuss off-PR.